workitem/simple_type.go

```diff
@@ -38,12 +38,12 @@ func (t SimpleType) GetKind() Kind {
 var timeType = reflect.TypeOf((*time.Time)(nil)).Elem()
 
 // ConvertToModel implements the FieldType interface
-func (fieldType SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
+func (t SimpleType) ConvertToModel(value interface{}) (interface{}, error) {
```

Comment:
Thank you that would be great and certainly makes the PR easier to review. Sorry that my initial question sounded a bit rude.